examples/oneview_enclosure.yml

```diff
@@ -93,41 +98,25 @@
         config: "{{ config }}"
         state: support_data_collection_set
         data:
-          name: 'Test-Enclosure-Renamed'
+          name: '{{ enclosure_name  }}'
           supportDataCollectionState: 'PendingCollection'
+      when: is_c7000 == true
       delegate_to: localhost
       ignore_errors: yes
       register: result
-
     - debug: var=result.msg
```

Comment:
This debug message doesn't make sense for the user if the previous task wasn't executed.
I suggest also add the conditional `when: is_c7000` here